common/config/src/env.rs

```diff
@@ -82,6 +82,10 @@ pub struct Config {
     #[example("none")]
     pub lookback: Option<String>,
 
+    #[env(LOGDNA_USE_K8S_API)]
```

Comment:
Can we name this something like

LOGDNA_K8S_METADATA_LOG_ENRICHMENT

or something slightly more descriptive? I realise that it's very long for a name, USE_K8S_API is quite vague